FlowCal/transform.py

```diff
@@ -118,6 +128,15 @@ def to_rfi(data, channels=None, max_range=None, amplification_type=None):
         the first element is zero, the amplification type is linear. If
         None, take `amplification_type` from
         ``data.amplification_type(channel)``.
```

Comment:
this description of `amplification_type` sounds pretty non-intuitive. might be nice to reference the FCS standards to indicate that it's actually just mimicking the `$PnE` keyword. this is minor, though.